DokanStore/DokanStore/XIB/CustomView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomView.swift
+//  DokanStore
+//
+//  Created by heba isaa on 18/06/2022.
+//
+
+import UIKit
+
+class CustomView: UIViewFromNib {
+
+    @IBOutlet weak var valueTF: UITextField!
+    @IBOutlet weak var titleTxt: UILabel!
+    
+    @IBInspectable var text:String?{
```

Comment:
```suggestion
    @IBInspectable var text: String? {
```
also check Swift guidelines for spacing
https://github.com/raywenderlich/swift-style-guide

Reply:
👍🏻👍🏻